workflows/customer_service_workflow/main.py

```diff
@@ -0,0 +1,186 @@
+import os
+import uuid
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+
+from swarmzero.agent import Agent
+from swarmzero.sdk_context import SDKContext
+from swarmzero.workflow import Workflow, WorkflowStep, StepMode
+
+CONFIG_PATH = os.path.join(os.path.dirname(__file__), "swarmzero_config.toml")
+sdk_context = SDKContext(config_path=CONFIG_PATH)
+
+# Agent 1: Greeting Agent
+greeting_agent = Agent(
+    name="greeting_agent",
+    functions=[],  # no tools needed for simple greeting
+    instruction="You are a friendly customer service representative. Greet customers warmly and ask how you can help them.",
+    sdk_context=sdk_context,
+    agent_id=str(uuid.uuid4()),
+    role="Customer Service Greeter",
+    description="Handles initial customer greetings and introductions"
+)
+
+# Agent 2: Issue Classification Agent
+def classify_issue_type(issue_description: str) -> str:
+    """Classify the type of customer issue based on description."""
+    # this could be a more sophisticated classification function
+    if any(word in issue_description.lower() for word in ["bill", "payment", "charge", "invoice"]):
+        return "billing"
+    elif any(word in issue_description.lower() for word in ["broken", "error", "not working", "technical"]):
+        return "technical"
+    elif any(word in issue_description.lower() for word in ["refund", "return", "cancel"]):
+        return "refund"
+    else:
+        return "general"
```

Comment:
this should be an agent itself. users could enter information that is relevant but doesn't include those keywords. an agent with proper instructions will figure this out on its own